examples/sql/sql_basic_query.cpp

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <hazelcast/client/hazelcast_client.h>
+
+int
+main()
+{
+    using namespace hazelcast::client::sql;
+
+    auto hz = hazelcast::new_client().get();
+
+    // populate the map with some data
+    auto map = hz.get_map("map").get();
```

Comment:
```suggestion
    auto map = hz.get_map("integer").get();
```

should not this match the mapping name?